modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -29,12 +29,40 @@ Use this option if you created your API outside API Manager, but want to apply p
 Use this option if you want to provide additional security for your API. 
 +
 . Based on the type of endpoint you are creating, specify the application type for a basic endpoint or the proxy deployment target for an endpoint with proxy.
-. If you want to manage the API using Mule 4 or later, select the *Mule version* checkbox.
-> In *Implementation URI*, replace the example by entering the URL of the site that hosts the API.
+. If you want to manage the API using Mule 4 or later, perform the following steps:
+.. Select the *Mule version* checkbox.
+.. In *Implementation URI*, replace the example by entering the URL of the site that hosts the API.
 +
 For example, you can use the URL of your API asset in Exchange. In this case, the API is consuming the JSONPlaceholder API, so enter `+http://jsonplaceholder.typicode.com+`.
+.. If you want to secure outbound traffic, click the elipses (*+*) button next the *Add TLS Context* link and specify the required details in the *Select TLS Context* window.
```

Comment:
Do you mean elipses (*...*) or do you mean plus symbol (*+*)?

You could also simplify this to either "Click *...*" or "Click *+*.

Reply:
Per accessibility guidelines, it is better to say both, right? Done! Thanks!